docs/Building a SvelteKit Demo Page with Web Component and Passkey Login using Corbado.com.md

```diff
@@ -0,0 +1,370 @@
+In this blog post, we'll be walking through the process of building a demo page for [passkeys.eu](https://passkeys.eu/) using SvelteKit. We'll cover how to create a reusable web component and implement passkey login functionality for a seamless user experience. This tutorial assumes basic familiarity with SvelteKit, HTML, CSS and JavaScript. Let's dive in!
```

Comment:
Finde ich gut, aber hier davor sowas wie "Introduction" packen?

Dann noch ggf. alles nummerieren in den Überschriften und ganz zu Beginn ein Inhaltsverzeichnis zeigen, sodass man sieht was kommt.